Src/Microsoft.Dynamic/Generation/MethodSignatureInfo.cs

```diff
@@ -37,9 +37,8 @@ public MethodSignatureInfo(MethodInfo info)
 
         public override bool Equals(object obj) {
             MethodSignatureInfo args = obj as MethodSignatureInfo;
-            if (args == null) return false;
 
-            if (args._isStatic != _isStatic || args._pis.Length != _pis.Length || args._genericArity != _genericArity) {
+            if (args?._isStatic != _isStatic || args._pis.Length != _pis.Length || args._genericArity != _genericArity) {
```

Comment:
Same as before (not a fan) although I find this case more confusing since it's an or.